template.json

```diff
@@ -1,13 +1,16 @@
 {
   "package": {
     "dependencies": {
+      "@reduxjs/toolkit": "^1.1.0",
```

Comment:
Let's at _least_ bump this to 1.2.5 (current).  We should probably do the same for the JS one too.